ipaserver/plugins/baseuser.py

```diff
@@ -18,15 +18,17 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import six
+from ldap.dn import dn2str, str2dn
```

Comment:
Respectively implement reverse() to `ipapython.dn.DN` if it is not already there